docs/docs/persistent-volumes.md

```diff
@@ -39,69 +43,77 @@ Additionally, you need to set a `residency` in order to tell Marathon to setup a
 }
 ```
 
-## Benefits of using Local Persistent Volumes
-
-- All resources needed to run tasks of your stateful service are dynamically reserved, thus ensuring the ability to relaunch the task on the same node using the same volume when needed.
-- You don't need constraints to pin a task to a particular agent where its data resides
-- You can still use constraints to specify distribution logic
-- Marathon lets you locate and destroy an unused persistent volume if you don't need it anymore
-
-## Prerequisites
+- `containerPath`: The path where your application will read and write data. This can currently be only relative and non-nested (`"data"`, but not `"/data"`, `"/var/data"` or `"var/data"`).
+- `mode`: The mode of the volume. Currently, `"RW"` is the only possible value.
```

Comment:
Yes, I've added a word or two. It's basically just the access mode – read/write.